conn.go

```diff
@@ -583,4 +583,17 @@ type Config struct {
 	// CAP_SYS_ADMIN are required), and most applications should leave this set
 	// to 0.
 	NetNS int
+
+	// DisableNSLockThread
```

Comment:
Nit: let's preserve the typical Go documentation style.

```
// DisableNSLockThread disables package netlink's default goroutine thread locking behavior.
// By default, the library will ...
```

And then you can continue on with what you have.

Reply:
ok